### PR TITLE
additional package source for newer mesa-libGL

### PR DESCRIPTION
**Pull request by fandan**
Resolve glxinfo errors on recent Fedora host version (39) in combination with Radeon GPU

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -21,6 +21,8 @@ ENV WINEDEBUG=${WINEDEBUG:-fixme-all}
 
 RUN dpkg --add-architecture i386 
 
+RUN echo "deb http://ftp.de.debian.org/debian sid main" >> /etc/apt/sources.list
```
Comment (netbrain) [inline Dockerfile]: Maybe use the official mirror? 🤷

Comment (netbrain) [reply Dockerfile:24]:
```suggestion
RUN echo "deb http://ftp.debian.org/debian sid main" >> /etc/apt/sources.list
```

**Review by netbrain — CHANGES_REQUESTED** [review]

**Comment by netbrain** [comment]
Tested it locally, seems to work fine. Releasing it to the wild to see what happens.

**Comment by gaeljw** [comment]
Worked fine for me, thanks folks!

(The issue described in comments of https://github.com/netbrain/zwift/issues/54 is solved :+1: )

**Pull request merged**